src/styles/index.scss

```diff
@@ -1,3 +1,213 @@
+// body {
+//   margin: 0;
+// }
+
+// .calendar {
+//   width: 1000px;
+//   height: 1000px;
+//   border: solid black;
+//   display: flex;
+// }
+
+// .day_1 {
+//   width: 100px;
+//   height: 100px;
+//   height: auto;
```

Comment:
There is a duplicate 'height' property in the '.day_1' class. The second 'height' property will override the first one. Please remove the redundant property.